example/example.cpp

```diff
@@ -26,9 +26,32 @@ void custom_flags_example();
 #include "spdlog/spdlog.h"
 #include "spdlog/cfg/env.h"  // support for loading levels from the environment variable
 #include "spdlog/fmt/ostr.h" // support for user defined types
+#include "spdlog/common.h"
+
+spdlog::file_event_handlers_t file_event_handlers;
+
+void init_file_event_handler() 
```

Comment:
please move this towards the end of example.cpp

Reply:
fixed.